.github/workflows/multi-node.yml

```diff
@@ -0,0 +1,63 @@
+name: Multi node test
+on:
+  # pull_request:
+  push:
+    # branches:
+    #   - master
+    # tags-ignore: [v.*]
+jobs:
+  run-multi-node-tests:
+    name: Multi Node Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix: # *DO NOT EDIT* the following line is generated by running ./scripts/gh-generate-projects-matrix.sh from the root folder
+        project: [akka-actor, akka-actor-testkit-typed, akka-actor-typed, akka-cluster, akka-cluster-metrics, akka-cluster-sharding, akka-cluster-sharding-typed, akka-cluster-tools, akka-cluster-typed, akka-coordination, akka-discovery, akka-distributed-data, akka-serialization-jackson, akka-multi-node-testkit, akka-osgi, akka-persistence, akka-persistence-query, akka-persistence-typed, akka-persistence-testkit, akka-protobuf, akka-protobuf-v3, akka-pki, akka-remote, akka-slf4j, akka-stream, akka-stream-testkit, akka-stream-typed, akka-testkit, akka-actor-tests, akka-actor-typed-tests, akka-bench-jmh, akka-docs, akka-bill-of-materials, akka-persistence-shared, akka-persistence-tck, akka-persistence-typed-tests, akka-remote-tests, akka-stream-tests, akka-stream-tests-tck]
+      fail-fast: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup SSH to local
+        run: |
+          mkdir -p ~/.ssh
+
+          # setup the authorized key
+          ssh-keygen -t ed25519 -a 100 -f ${HOME}/.ssh/id_rsa -q -N ""
+          sudo ssh-keyscan -H 127.0.0.1 >> ${HOME}/.ssh/known_hosts
+          sudo ssh-keyscan -H localhost >> ${HOME}/.ssh/known_hosts
+          sudo cat ${HOME}/.ssh/id_rsa.pub >> ${HOME}/.ssh/authorized_keys
+
+          # fix the filesystem permissions
+          sudo chmod go-w ~/
+          sudo chmod 700 ~/.ssh
+          sudo chmod 600 ~/.ssh/authorized_keys
+
+          # restart the ssh daemon
+          sudo service sshd restart
+
+          ssh runner@127.0.0.1 uptime
+      - name: Set up JDK 11
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11.0-9
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.2
+      - name: Multi node test
+        env:
+          JOB_ID: ${{ github.run_id }}
+        run: |
+          sbt -jvm-opts .jvmopts-ci \
+            -Dakka.test.timefactor=1 \
+            -Dakka.cluster.assert=on \
+            -Dsbt.override.build.repos=false \
+            -Dakka.test.multi-node=true \
+            -Dakka.test.multi-node.targetDirName=${PWD}/target/${JOB_ID} \
+            -Dakka.test.multi-node.java=${JAVA_HOME}/bin/java \
+            -Dmultinode.XX:MetaspaceSize=128M \
+            -Dmultinode.XX:+UseCompressedOops \
+            -Dmultinode.Xms256M \
+            -Dmultinode.Xmx512M \
+            -Dmultinode.host=127.0.0.1 \
+            test:compile \
+            ${{ matrix.project }}/test
```

Comment:
doesn't this run the 'regular' tests as well? I'd be OK with not testing those here.